common4j/src/main/com/microsoft/identity/common/java/util/ResultFuture.java

```diff
@@ -58,7 +58,7 @@ public boolean isDone() {
     }
 
     @Override
-    public synchronized T get() throws InterruptedException, ExecutionException {
+    public T get() throws InterruptedException, ExecutionException {
```

Comment:
Why are we removing the synchronized keyword from these methods?

Reply:
I shouldn't have added it. This function calls await and would cause a deadlock with the synchronized keyword.

Removing it here means the code is back where it was before this PR.

Reply:
You know, I think I recall this.  Should probably add documentation so that nobody else tries it.